pkg/cmd/artefactor.go

```diff
@@ -29,6 +30,14 @@ const (
 	// FlagImageVars is used to specify a whitelist of variable names to enable
 	// "export"
 	FlagImageVars = "image-vars"
+	// FlagDockerUserName overrides docker registry configuration
+	FlagDockerUserName = "docker-username"
+	// FlagDockerPassword overrides docker registry configuration
+	FlagDockerPassword = "docker-password"
+	// FlagDockerPasswordHelp is displayed when getting help for the flag
+	FlagDockerPasswordHelp = "overrides docker registry configuration for password"
+	// FlagDockerPasswordHelp is displayed when getting help for the flag
```

Comment:
Comment should be `FlagDockerUserNameHelp`

Reply:
Indeed - added #70

Reply:
nice - not sure why linter didn't catch - added #70